galette/templates/default/pages/configuration_dynamic_fields.html.twig

```diff
@@ -1,13 +1,13 @@
 {% extends 'page.html.twig' %}
 
 {% block content %}
-    <div class="ui three item top attached stackable tabs menu">
+    <div class="ui stackable pointing inverted menu tabbed">
 {% for key, form in all_forms %}
-        <a href="{{ path_for('configureDynamicFields', {'form_name': key}) }}" class="item{% if form_name == key %} active{% endif %}" data-tab="{{ loop.index }}">{{ form }}</a>
+        <a href="{{ path_for('configureDynamicFields', {'form_name': key}) }}" class="item{% if form_name == key %} active{% endif %}" data-tab="{{ key }}">{{ form }}</a>
 {% endfor %}
     </div>
 {% for key, form in all_forms %}
-    <div class="ui bottom attached{% if form_name == key %} active{% endif %} tab segment" data-tab="{{ loop.index }}">
+    <div class="ui bottom attached{% if form_name == key %} active{% endif %} tab segment" data-tab="{{ key }}">
```

Comment:
```suggestion
    <div class="ui{% if form_name == key %} active{% endif %} tab segment" data-tab="{{ key }}">
```
Segments cannot be attached to pointing menus. This is the reason why some tabs are missing a top border.